test/index.test.js

```diff
@@ -104,4 +104,25 @@ describe('gulp-ejs', function() {
 
     stream.end()
   })
+
+  it('should render async ejs template', function(done) {
+    const title = () => new Promise((resolve, reject) => {
+      process.nextTick(() => resolve('gulp-ejs'))
+    })
+    const stream = ejs({ title: title }, { async: true })
+
+    stream.on('data', data => {
+      assert.strictEqual(data.contents.toString(), '<h1>gulp-ejs</h1>')
+    })
+
+    stream.on('end', done)
+
+    stream.write(
+      new Vinyl({
+        contents: Buffer.from('<h1><%= await title() %></h1>')
```

Comment:
Although it's deprecated, I would really like to maintain full compatibility with Node 6 (this is what's currently breaking the travis build). Can we change this to support Node 6?

Reply:
I added `this.skip()` if async function is not supported.